tests/networkloadbalancer/vars.yml

```diff
@@ -7,7 +7,7 @@ lan: "AnsibleAutoTestNLB"
 location: gb/lhr
 image: ce08ca15-0adf-11ec-9fa6-1e5ea9024c03
 image_alias: centos:7
-password: qq6mdvirWwj1dsdXjcpp
+password: "{{ lookup('ansible.builtin.password', '/dev/null chars=ascii_letters,digits,punctuation') }}"
 updated_password: fjee83jkw940djcer
```

Comment:
looks like there is also a field here which needs the lookup :+1:

Reply:
added